torch/linalg/__init__.py

```diff
@@ -1228,8 +1228,6 @@
 
 Computes a vector or matrix norm.
 
-If :attr:`A` is complex valued, it computes the norm of :attr:`A`\ `.abs()`
```

Comment:
Why remove this?

Reply:
I realised this is not quite right. This is fine for vector norms (for which it is already documented, as we say that it computes `abs(x)` first in the formulas) but it is not true for the matrix 2-norm (largest singular value).